types/mojang-gametest/index.d.ts

```diff
@@ -841,7 +851,7 @@ export class SimulatedPlayer extends mojangminecraft.Player {
  * These well-known tags can be used to classify different
  * tests into suites to run.
  */
-// tslint:disable-next-line:no-unnecessary-class
+ // tslint:disable-next-line:no-unnecessary-class
```

Comment:
This change seems to have been made all over the place; was this intentional?

Reply:
@jakebailey not intentional.  Let me double check our pipeline.  These files are autogenerated so I will check the spacing on our side.  Thanks for pointing this out!